### PR TITLE
RTL Support added for 17+ devices

### PR DESCRIPTION
**Pull request by shubhamtewari**
added support for RTL for 17+ devices

### CODE REVIEW

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java [hunk 1]
```diff
@@ -452,9 +452,9 @@ public void onClick(View v) {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
             if (config.getLayoutDirection() == View.LAYOUT_DIRECTION_RTL) {
                 sendButton.setScaleX(-1);
-                mainEditTextLinearLayout.setBackgroundResource(R.drawable.applozic_chat_left_icon);
-                audioRecordFrameLayout.setBackgroundResource(R.drawable.applozic_chat_left_icon);
-                slideImageView.setImageResource(R.drawable.slide_arrow_right);
+                //mainEditTextLinearLayout.setBackgroundResource(R.drawable.applozic_chat_left_icon);
+                //audioRecordFrameLayout.setBackgroundResource(R.drawable.applozic_chat_left_icon);
+                //slideImageView.setImageResource(R.drawable.slide_arrow_right);
```
Comment (reytum) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java]: remove comments

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/richmessaging/KmCustomLayoutManager.java [hunk 2]
```diff
@@ -11,18 +12,21 @@
 
     public KmCustomLayoutManager(Context context) {
         super(context);
+        Log.d("rtl", "const called");
     }
 
     public KmCustomLayoutManager(Context context, AttributeSet attrs) {
-        super(context, attrs);
+        super(context, attrs);Log.d("rtl", "const called");
```
Comment (reytum) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/richmessaging/KmCustomLayoutManager.java]: remove Log.d

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/views/KmRecordButton.java [hunk 3]
```diff
@@ -6,6 +6,7 @@
 import android.support.v7.content.res.AppCompatResources;
 import android.support.v7.widget.AppCompatImageView;
 import android.util.AttributeSet;
+import android.util.Log;
```
Comment (reytum) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/views/KmRecordButton.java]: remove this import

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/views/KmRecordView.java [hunk 4]
```diff
@@ -2,13 +2,15 @@
...
 import android.content.res.AssetFileDescriptor;
+import android.content.res.Configuration;
 import android.content.res.TypedArray;
 import android.graphics.drawable.Drawable;
 import android.media.MediaPlayer;
 import android.os.SystemClock;
 import android.support.annotation.Nullable;
 import android.support.v7.content.res.AppCompatResources;
 import android.util.AttributeSet;
+import android.util.Log;
```
Comment (reytum) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/views/KmRecordView.java]: remove this

File: kommunicateui/src/main/res/layout/mobicom_sent_message_list_view.xml [hunk 5]
```diff
@@ -101,7 +102,7 @@
                 android:id="@+id/createdAtTime"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_gravity="right|end|bottom"
+                android:layout_gravity="end|end|bottom"
```
Comment (reytum) [inline kommunicateui/src/main/res/layout/mobicom_sent_message_list_view.xml]: remove one end

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java (hunk 1 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java]: done

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/richmessaging/KmCustomLayoutManager.java (hunk 2 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/richmessaging/KmCustomLayoutManager.java]: done

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/views/KmRecordButton.java (hunk 3 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/views/KmRecordButton.java]: done

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/views/KmRecordView.java (hunk 4 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/views/KmRecordView.java]: done

File: kommunicateui/src/main/res/layout/mobicom_sent_message_list_view.xml (hunk 5 above)
Comment (shubhamtewari) [reply kommunicateui/src/main/res/layout/mobicom_sent_message_list_view.xml]: done

**Pull request closed without merging**